src/bosh_openstack_cpi/spec/integration/lifecycle_spec.rb

```diff
@@ -243,6 +243,99 @@ def where_ip_address_is(ip)
     end
   end
 
+  context 'when booting from volume with a boot_volume_type' do
+    let(:boot_from_volume) { true }
+    let(:boot_volume_type) { @boot_volume_type }
```

Comment:
I don't think you can specify `boot_volume_type` anymore. I also don't see this used in the tests. Can we remove the `boot_volume_type` stuff?